### PR TITLE
SFI-Image fix part 1

### LINKED ISSUE #33747: SFI - Images - sev 1 - aspnetcore.docs
Fix all sev 1 SFI images for aspnetcore.docs

---
[Associated WorkItem - 319452](https://dev.azure.com/msft-skilling/Content/_workitems/edit/319452)

### PR DESCRIPTION
**Pull request by wadepickett**
Fixes #33747   (Sev 0-1 images)

Reviewed all images, there was only two sev 0-1 to fix:
/aspnetcore/security/authentication/social/index/_static/FBProductSetup.png
/aspnetcore/tutorials/publish-to-azure-webapp-using-vs/_static/pp_settings.png

Fixed that but also fixed a few in the same general area while I was there.

### CODE REVIEW

File: aspnetcore/security/authentication/social/index/_static/pass1a.png
Comment (tdykstra) [inline aspnetcore/security/authentication/social/index/_static/pass1a.png:1]: I don't see what changed in this one.

File: aspnetcore/security/authentication/social/index/_static/FBProductSetup.png
Comment (tdykstra) [inline aspnetcore/security/authentication/social/index/_static/FBProductSetup.png:1]: I don't see what changed in this one.

**Review by tdykstra — APPROVED** [review]
LGTM, a couple comments.

File: aspnetcore/security/authentication/social/index/_static/pass1a.png
Comment (wadepickett) [reply aspnetcore/security/authentication/social/index/_static/pass1a.png:1]: email address in top right changed from actual email address to "<email address>".  Hard to see it, pretty small and grey, but it is what was called out in the scan.

File: aspnetcore/security/authentication/social/index/_static/FBProductSetup.png
Comment (wadepickett) [reply aspnetcore/security/authentication/social/index/_static/FBProductSetup.png:1]: email address on top right was called out so it was removed.

**Pull request merged**